modules/rosapps/applications/sysutils/man/man.c

```diff
@@ -88,33 +88,40 @@ Usage()
 int
 AnalyzeArgv(char *argument)
 {
-    int element=0;
+    int element;
     char HelpFlag=0;
     char *keys[]={"--help","/h","/?","-h"};
     char *sections[]={".1",".2",".3",".4",".5",".6",".7",".8",".9"};
     char *filename=(char*)malloc(sizeof(char)*MAXLINE);
 
     strcpy(filename,argument); //save argument value
 
-    for(element=0; element < 5;element++)
+    for(element = 0; element < 4; element++)
```

Comment:
Can you try using `ARRAYSIZE(keys)` or `_countof(keys)` ?

Reply:
Switched to `_countof()`.